src/participant.ts

```diff
@@ -284,6 +289,7 @@ export class Participant extends EventEmitter {
     method: 'verificationChallenge',
     fn: (params: { challenge: string }) => Promise<string>,
   ): this;
+  public add(method: string, fn: (params: any) => any): this;
```

Comment:
What's the difference between this signature and the next line

Reply:
Check out the docs at the bottom of this page: https://www.typescriptlang.org/docs/handbook/functions.html

In typescript, function overloads hide the actual implementation - so we have to add an overload with the same signature for it to be callable (or rather, for the types to resolve - it's always callable)